frontend/src/views/housing/project/ProjectView.vue

```diff
@@ -0,0 +1,384 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { computed, onMounted, ref } from 'vue';
+import { useRouter } from 'vue-router';
+
+import { Accordion, AccordionTab, Button, Card, Divider } from '@/lib/primevue';
+import { RouteName } from '@/utils/enums/application';
+import { PermitAuthorizationStatus, PermitNeeded, PermitStatus } from '@/utils/enums/housing';
+import { formatDate } from '@/utils/formatters';
+
+import { permitService, submissionService, userService } from '@/services';
+import { useSubmissionStore, useTypeStore } from '@/store';
+
+import type { ComputedRef, Ref } from 'vue';
+import type { Permit, PermitType, User } from '@/types';
+
+const router = useRouter();
+
+type PermitFilterConfig = {
+  permitNeeded?: string;
+  permits: Array<Permit>;
+  permitStatus?: string;
+  permitTypes: Array<PermitType>;
+};
+
+type CombinedPermit = Permit & PermitType;
+
+// Props
+type Props = {
+  activityId: string;
+  submissionId: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// State
+const submissionStore = useSubmissionStore();
+const { getPermits, getSubmission } = storeToRefs(submissionStore);
+
+const typeStore = useTypeStore();
+const { getPermitTypes } = storeToRefs(typeStore);
+
+const assignee: Ref<User | undefined> = ref(undefined);
+const loading: Ref<boolean> = ref(true);
+const permitsNeeded = computed(() => {
+  return permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.NEW,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  })
+    .sort(permitNameSortFcn)
+    .sort(permitBusinessSortFcn);
+});
+const permitsNotNeeded = computed(() => {
+  return permitFilter({
+    permitNeeded: PermitNeeded.NO,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  })
+    .sort(permitNameSortFcn)
+    .sort(permitBusinessSortFcn);
+});
+const permitsSubmitted: ComputedRef<Array<CombinedPermit>> = computed(() => {
+  let firstFilter = permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.APPLIED,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  });
+  let secondFilter = permitFilter({
+    permitNeeded: PermitNeeded.YES,
+    permitStatus: PermitStatus.COMPLETED,
+    permits: getPermits.value,
+    permitTypes: getPermitTypes.value
+  });
+  return firstFilter.concat(secondFilter).sort(permitNameSortFcn).sort(permitBusinessSortFcn);
+});
```

Comment:
Possible bug here? I am seeing the same permits under Required and Submitted. 
https://pcns-dev-pr-122.apps.silver.devops.gov.bc.ca/housing/project?activityId=F734DE99&submissionId=649d5bb8-6d02-4361-bb5c-3949e9782aba

Reply:
Something got duplicated wrong. Fixed now.